tests/test_export_db.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2022 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+import sys
+import os
+
+from tuna.dbBase.sql_alchemy import DbSession
+from tuna.miopen.utils.config_type import ConfigType
+from tuna.miopen.db.tables import MIOpenDBTables
+from tuna.miopen.subcmd.export_db import get_fdb_query, build_miopen_fdb, get_pdb_query, build_miopen_kdb, export_fdb, get_filename
+from tuna.utils.db_utility import DB_Type
+from utils import add_test_session, DummyArgs
+
+sys.path.append("../tuna")
+sys.path.append("tuna")
+
+this_path = os.path.dirname(__file__)
+
+
+def test_export_db():
```

Comment:
I believe we can merge both testing files into one without removing much code...My tests are more focused on individual functional runs and callability while your test_export_db is more integrative and focused on ensuring that all functions run correctly relative to each others with the correct data parsing.